src/land/imsfv3_scf2ioda.py

```diff
@@ -84,14 +84,17 @@ def _read(self):
         ncd = nc.Dataset(self.filename)
         lons = ncd.variables['lon'][:].ravel()
         lats = ncd.variables['lat'][:].ravel()
+        stid = ncd.variables['stid'][:].ravel()
         oros = ncd.variables['oro'][:].ravel()
         sncv = ncd.variables['IMSscf'][:].ravel()
         sncv[sncv == -999.] = float_missing_value
         sndv = ncd.variables['IMSsnd'][:].ravel()
         sndv[sndv == -999.] = float_missing_value
+        strstid = np.empty_like(lons, dtype=object)
 
         lons = lons.astype('float32')
         lats = lats.astype('float32')
+        stid = stid.astype('int64')
```

Comment:
stid = stid.astype('int32')

Reply:
@yuanxue2870 I recalled that only datetime use the double precision. For station ID, single precision is sufficient.

Reply:
Okay, keep it as suggested, @yuanxue2870. I am fine with it but this is the first time I found that people uses int64 for station ID, a bit weird.

Reply:
That is an optimal suggestion, @srherbener, thank you for your deep thoughts. sounds good with me.